### ISSUE TITLE
ES6 JS tree shaking not working.

### ISSUE THREAD

**Issue by jasan-s** [ant-design/ant-design#17496]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/jasan-s/react-embeddable-widget-starter/tree/antd](https://github.com/jasan-s/react-embeddable-widget-starter/tree/antd)

### Steps to reproduce
`npm run build`

notice the build size gzip `209.7 KB`. 
if i remove the 
```
import Button from 'antd/es/button';
<Button type="primary">Button</Button>
```
the size of built bundle is gzip `37.66 KB`

### What is expected?
expected built index.js to be increase by approx.  `14kb` in size. without antd the index.js is  gzip `37.66 KB` so after build it should be approx. gzip `50kb`.

### What is actually happening?
the built index.js is  showing gzip`209.7 KB` in size. leading me to believe the tree shaking is not working for JS antd files.

<img width="1680" alt="Screen Shot 2019-07-05 at 7 11 16 PM" src="https://user-images.githubusercontent.com/13863953/60750119-c73b4700-9f58-11e9-909c-6f23184c0844.png">

| Environment | Info |
|---|---|
| antd | 3.20.0 |
| React | 16.8.6 |
| System | mac osx |
| Browser | chrome |

### UPDATE:
Also tested it with `create-react-app@3.0.1` tree shaking not working. 
Steps to reproduce: 
1: create new app `create-react-app treeshakeTest`
2: `cd treeshakeTest` 
3: npm i --save antd
4: in `src/App.js` add import Button from 'antd/es/button' && 
`<Button type="primary">Button</Button>` somewhere in the return

Result: with the `antd` import the bundle increases by +171.28 KB see image 
<img width="603" alt="Screen Shot 2019-07-05 at 11 23 09 PM" src="https://user-images.githubusercontent.com/13863953/60752561-eb5c4f80-9f7b-11e9-9cba-a85e0edd2a1f.png">

Prior to the `antd` import:
<img width="510" alt="Screen Shot 2019-07-05 at 11 24 28 PM" src="https://user-images.githubusercontent.com/13863953/60752580-19da2a80-9f7c-11e9-8a79-3470ddbbe41d.png">

also posted it on https://stackoverflow.com/questions/56911819/es6-js-tree-shaking-not-working-for-antd-lib

**Comment by afc163** [comment]
Duplicate of #12011

**Comment by RewriteH** [comment]
> Duplicate of #12011

men its not duplicate bc #12011 issue the answers are only in Chinese...

**Comment by yoyo837** [comment]
It's fixed in v4.0, please update to version 4.0.
